api_tests/nodes/views/test_node_preprints.py

```diff
@@ -42,43 +43,54 @@ def setUp(self):
         self.url = '/{}nodes/{}/preprints/?version=2.2&'.format(API_BASE, self.published_project._id)
         super(TestNodePreprintIsPublishedList, self).setUp()
 
-class TestNodePreprintIsValidList(PreprintIsValidListMixin, ApiTestCase):
-    def setUp(self):
-        self.admin = AuthUserFactory()
-        self.provider = PreprintProviderFactory()
-        self.project = ProjectFactory(creator=self.admin, is_public=True)
-        self.url = '/{}nodes/{}/preprints/?version=2.2&'.format(API_BASE, self.project._id)
-        super(TestNodePreprintIsValidList, self).setUp()
+class TestNodePreprintIsValidList(PreprintIsValidListMixin):
+    @pytest.fixture()
+    def admin(self):
```

Comment:
I have a recollection of Steve wanting either fixtures or the mixin style since it is a bit odd to have both. You might need to ask him for clarification on how he wanted this done as I don't remember.